benchmark/bench-runner/Main.hs

```diff
@@ -35,22 +35,6 @@ rtsOpts exeName benchName0 = unwords [general, exeSpecific, benchSpecific]
              `isPrefixOf` benchName = "-K2M -M256M"
         | "Prelude.Rate/o-1-space." `isPrefixOf` benchName = "-K128K"
         | "Prelude.Rate/o-1-space.asyncly." `isPrefixOf` benchName = "-K128K"
-        | "Prelude.Serial/o-1-space.mixed.sum-product-fold" == benchName =
-            "-K64M"
-        | "Prelude.Serial/o-n-heap.grouping.classifySessionsOf"
-            `isPrefixOf` benchName = "-K1M -M32M"
-        | "Prelude.Serial/o-n-heap.Functor." `isPrefixOf` benchName =
-            "-K4M -M32M"
-        | "Prelude.Serial/o-n-heap.transformer." `isPrefixOf` benchName =
-            "-K8M -M64M"
-        | "Prelude.Serial/o-n-space.Functor." `isPrefixOf` benchName =
-            "-K4M -M64M"
-        | "Prelude.Serial/o-n-space.Applicative." `isPrefixOf` benchName =
-            "-K8M -M128M"
-        | "Prelude.Serial/o-n-space.Monad." `isPrefixOf` benchName =
-            "-K8M -M64M"
-        | "Prelude.Serial/o-n-space.grouping." `isPrefixOf` benchName = ""
-        | "Prelude.Serial/o-n-space." `isPrefixOf` benchName = "-K4M"
```

Comment:
Don't we need to add these options for the new Data.Stream benchmarks? Otherwise those will fail to run. Ideally we just need to replace Prelude.Serial with Data.Stream in the code above.